x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/linux_shell_activity_by_web_server.json

```diff
@@ -7,9 +7,9 @@
     "auditbeat-*"
   ],
   "language": "kuery",
-  "max_signals": 33,
+  "max_signals": 100,
   "name": "Potential Shell via Web Server",
-  "query": "process.name: bash and (user.name: apache or www) and event.action:executed",
+  "query": "process.name: bash and user.name: (apache or www or \"wwww-data\") and event.action:executed",
```

Comment:
@FrankHassanabad @randomuserid should this be `www-data`, not `wwww-data`?

Reply:
Yes that is a typo and IDK how that got through. IDK if we have any options at this point but either fixing it as suggested or reverting to the earlier search version without the www-data user would seem valid. If it is not too late. Thanks very much for catching this!